scripts/python/cpplint.py

```diff
@@ -746,7 +746,7 @@ def IsErrorSuppressedByNolint(category, linenum):
     category: str, the category of the error.
     linenum: int, the current line number.
   Returns:
-    bool, True iff the error should be suppressed due to a NOLINT comment or
+    bool, True if the error should be suppressed due to a NOLINT comment or
```

Comment:
The "iff" might have been on purpose: https://en.wikipedia.org/wiki/If_and_only_if